fastify.js

```diff
@@ -4,6 +4,7 @@ const VERSION = '4.10.2'
 
 const Avvio = require('avvio')
 const http = require('http')
+const deepClone = require('rfdc')({ circles: true, proto: false })
```

Comment:
We should not have cycle deps to deal with

```suggestion
const deepClone = require('rfdc')({ circles: false, proto: false })
```

Reply:
many tnks for your suggestion
but unfortunately it is not enough... I'm investigating